js/pihole/index.js

```diff
@@ -13,24 +13,6 @@ $(document).ready(function() {
 
     updateTopLists();
```

Comment:
All of these functions, besides `updateSummaryData`, might be better after we create the charts, just to be sure the charts exist before we access them.